exchanges/orderbook/validate.go

```diff
@@ -0,0 +1,96 @@
+package orderbook
+
+import (
+	"fmt"
+
+	"github.com/thrasher-corp/gocryptotrader/log"
+)
+
+// checker defines specific functionality to determine ascending/descending
+// validation
```

Comment:
🐒 Given L30, I really can't process the rules your editor is following for wrapping 😂 
```suggestion
// checker defines specific functionality to determine ascending/descending validation
```